src/main/java/com/rockset/client/api/CustomDocumentsApi.java

```diff
@@ -89,21 +86,28 @@ Object buildRocksetObject(Object doc) {
       // HH:mm:ss.SSSSSSSSS
       return constructTime(((LocalTime) doc).toString());
     } else if (doc instanceof Timestamp) {
-      // Timestamp toString formats to yyyy-mm-dd hh:mm:ss.fffffffff format,
-      // so to use uuuu-MM-dd'T'HH:mm:ss.SSSSSSSSS notation, convert
-      // to LocalDateTime first
-      return constructDateTime(((Timestamp) doc).toLocalDateTime().toString());
+      return constructTimestamp(((Timestamp) doc).toInstant());
```

Comment:
changing this is makes sense as we didn't release this code, so no backwards compatibility concerns